omnisend-for-formidable-forms-add-on/includes/Templates/header.php

```diff
@@ -0,0 +1,54 @@
+<?php
+	$setting_code = $this->get_enabled_code_by_form_id( $values['id'] );
```

Comment:
Can you add anotation/comment of type $this, so IDE can recognise what class it is.